src/tests/components/LocationDetails.test.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LocationDetails from "../../components/LocationDetails";
+
+describe("LocationDetails", () => {
+  it("renders the correct city and location props", () => {
```

Comment:
Could another test be added here to check if the error message is displayed if it is passed in via some valid props?